deployments/localhost/usdc/roots.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We should delete this; no need to commit an empty roots file

Reply:
Done